libcxx/utils/generate_libcxx_cppm_in.py

```diff
@@ -42,33 +44,46 @@
 // The headers of Table 24: C++ library headers [tab:headers.cpp]
 // and the headers of Table 25: C++ headers for C library facilities [tab:headers.cpp.c]
 """
-    )
-    for header in module_headers:
-        if header in header_restrictions:
-            std_module_cpp_in.write(
-                f"""\
+        )
+        for header in module_headers:
+            if header in header_restrictions:
+                module_cpp_in.write(
+                    f"""\
 #if {header_restrictions[header]}
 #  include <{header}>
 #endif
 """
-            )
-        else:
-            std_module_cpp_in.write(f"#include <{header}>\n")
+                )
+            else:
+                module_cpp_in.write(f"#include <{header}>\n")
 
-    std_module_cpp_in.write("\n// *** Headers not yet available ***\n")
-    for header in sorted(headers_not_available):
-        std_module_cpp_in.write(
-            f"""\
+        module_cpp_in.write("\n// *** Headers not yet available ***\n")
+        for header in sorted(headers_not_available):
+            module_cpp_in.write(
+                f"""\
 #if __has_include(<{header}>)
-#  error "update the header information for <{header}> in libcxx/utils/generate_std_cppm_in.py"
-#endif //   __has_include(<{header}>)
+#  error "update the header information for <{header}> in headers_not_available in utils/libcxx/header_information.py"
```

Comment:
```suggestion
#  error "please update the header information for <{header}> in headers_not_available in utils/libcxx/header_information.py"
```

This makes it clearer that we're asking the developer to do this. Or we could also say `the header information in <...> needs to be updated.`